schemas/context/enduserids.schema.json

```diff
@@ -44,5 +44,5 @@
       "$ref": "https://ns.adobe.com/xdm/context/identity"
     }
   },
-  "meta:status": "experimental"
+  "meta:status": "stabilizing"
```

Comment:
There is an inconsistency between `target` and `tnt` IDs in this schema. 👎

Reply:
Good catch @trieloff, I posted a PR to fix this.

Reply:
merged. Therefore: 👍